mkosi/machine.py

```diff
@@ -155,6 +160,12 @@ def __exit__(self, *args: Any, **kwargs: Any) -> None:
         self.stack.__exit__(*args, **kwargs)
 
 
+pytestmark = [
+    pytest.mark.integration,
+    pytest.mark.skipif(os.getuid() != 0, reason="Must be invoked as root.")
+]
+
+
```

Comment:
We shouldn't put pytest specifics in the machine.py file. We should instead make sure we do the same pytest check in MkosiMachineTest as well so that we skip tests when uid != 0 even if pytest isn't installed.